ui/admin/.template-lintrc.js

```diff
@@ -19,6 +18,8 @@ module.exports = {
       ],
     },
     quotes: 'single',
-    'eol-last': 'editorconfig',
+    'self-closing-void-elements': false,
+    'block-indentation': false,
+    'no-route-action': false,
```

Comment:
This is a new [rule](https://github.com/ember-template-lint/ember-template-lint/blob/v3.12.0/docs/rule/no-route-action.md) that actually discourages the use of `ember-route-action-helper` in place of just using controllers. 

I had to disable this as we use it everywhere but I added a topic to our tech time to discuss if we should migrate off of this.

Reply:
AMAZING!!! I love the way to managed this and bring it to FE tech time, Thanks!